lib/DatabaseFunctions.ts

```diff
@@ -0,0 +1,23 @@
+import Database from './Database'
```

Comment:
Instead of a file called `DatabaseFunctions`, do the following:
1. Create a folder in lib: `/lib/controllers`
2. Create a file specifically for notes: `/lib/controllers/NoteController.ts`
3. Encapsulate all note-related database functions in a class that inherits from the Database class:
```ts
class NoteController extends Database {
  construct() {
    super() // initialize the Database superclass
  }

  // Your functions...
}
```

Now you don't have to pass DB as a variable. You can just call Database class member functions directly.
Do the same for all other database schemas: e.g. a controller for chat history, for edges, etc.